src/libtrx/game/console/cmd/kill.c

```diff
@@ -42,7 +42,7 @@ static bool M_KillSingleEnemyInRange(const int32_t max_dist)
     int16_t best_item_num = NO_ITEM;
     for (int16_t item_num = 0; item_num < Item_GetTotalCount(); item_num++) {
         const ITEM *const item = Item_Get(item_num);
-        if (Creature_IsHostile(item)) {
+        if (Creature_IsHostile(item) && item->object_id != O_LARA) {
```

Comment:
Would you like me to include this in #3493 as an extra guard?

Reply:
Oh shit, it's a leftover of my own testing. I'll revert that